types/office-js-preview/index.d.ts

```diff
@@ -84156,6 +84156,56 @@ declare namespace Word {
          * @param propertyNamesAndPaths `propertyNamesAndPaths.select` is a comma-delimited string that specifies the properties to load, and `propertyNamesAndPaths.expand` is a comma-delimited string that specifies the navigation properties to load.
          */
         load(propertyNamesAndPaths?: OfficeExtension.LoadOption): Word.CommentCollection;
+        /**
+         * Occurs when new comments are added.
+         *
+         * @remarks
+         * [Api set: WordApiOnline BETA (PREVIEW ONLY)]
+         *
+         * @eventproperty
+         * @beta
+         */
+        readonly onCommentAdded: OfficeExtension.EventHandlers<Word.CommentEventArgs>;
+        /**
+         * Occurs when a comment or its reply is changed.
+         *
+         * @remarks
+         * [Api set: WordApiOnline BETA (PREVIEW ONLY)]
+         *
+         * @eventproperty
+         * @beta
+         */
+        readonly onCommentChanged: OfficeExtension.EventHandlers<Word.CommentEventArgs>;
+        /**
+         * Occurs when comments are deleted.
+         *
+         * @remarks
+         * [Api set: WordApiOnline BETA (PREVIEW ONLY)]
+         *
+         * @eventproperty
+         * @beta
+         */
+        readonly onCommentDeleted: OfficeExtension.EventHandlers<Word.CommentEventArgs>;
+        /**
+         * Occurs when a comment is deselected.
```

Comment:
This is the last one I'll explicitly call out, but be sure throughout that we're saying "comment or reply" if that's what's meant.